index.html

```diff
@@ -0,0 +1,199 @@
+                <ul class="list">
+                    <li><a href="#" class="button"><img src="images/html.png" alt=".html"></a></li>
+                    <li><a href="#" class="button"><img src="images/css.png" alt=".css"></a></li>
+                    <li><a href="#" class="button"><img src="images/javascript.png" alt=".js"></a></li>
+                </ul>
+                <a href="#" class="button"><img src="images/see-project.png" alt="Seeproject"></a>
+            </div>
+        </section>
+    </div>
+    <div class="card">
+        <section class="profile-picture">
+            <img src="images/snapshort-profile-4.png" alt="Snapshot-profile">
+        </section>
+        <section class="tonic">
+            <h1>Multi-Post<br>Stories</h1>
+            <div class="list">
+                <ul class="list">
+                    <li><p>CANOPY</p></li>
+                    <li><img src="images/counter-1.png" alt="counter1"></li>
+                    <li><p>Back End Dev</p></li>
+                    <li><img src="images/counter-2.png" alt="counter2"></li>
+                    <li><p>2015</p></li>
+                </ul>
+            </div>
+            <p class="contents">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <br>
+            <div class="buttons">
+                <ul class="list">
+                    <li><a href="#" class="button"><img src="images/html.png" alt=".html"></a></li>
+                    <li><a href="#" class="button"><img src="images/css.png" alt=".css"></a></li>
+                    <li><a href="#" class="button"><img src="images/javascript.png" alt=".js"></a></li>
+                </ul>
+                <a href="#" class="button"><img src="images/see-project.png" alt="Seeproject"></a>
+            </div>
+        </section>
+    </div>
+    
+    <div class="white-bg btl-rad">
+        <section class="about">
+            <br>
+            <h1 class="main-text">About Myself</h1>
+            <p class="content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me..</p>
+            <br>
+            <p>LET'S CONNECT</p>
+            <br>
+            <div class="contact">
+                <ul class="list">
+                    <li><a href="#"><img src="images/vector-1.png" alt="vector1"></a></li>
+                    <li><a href="#"><img src="images/vector-2.png" alt="vector2"></a></li>
+                    <li><a href="#"><img src="images/vector-3.png" alt="vector3"></a></li>
+                    <li><a href="#"><img src="images/vector-4.png" alt="vector4"></a></li>
+                    <li><a href="#"><img src="images/vector-5.png" alt="vector5"></a></li>
+                </ul>
+                <a href="#" class="button"><img src="images/get-my-resume.png" alt="Seeproject"></a>
```

Comment:
- [ ] I recommend you not to use an image of `Get Resume`. Instead, it would be better to use text instead.

- [ ] After changing this to a text, add the necessary styling required as indicated in the template. Active, Hover and Pressed.